plutus-core/testlib/PlutusCore/Generators/Internal/Dependent.hs

```diff
@@ -27,6 +29,8 @@ liftOrdering LT = GLT
 liftOrdering EQ = error "'liftOrdering': 'Eq'"
 liftOrdering GT = GGT
 
+type KnownType val a = (KnownTypeAst (UniOf val) a, MakeKnown val a)
```

Comment:
Naming here doesn't make much sense, but I really don't want to spend a lot of time here refactoring these tests, at the very least not in this PR.